resources/scss/_common.scss

```diff
@@ -0,0 +1,321 @@
+@import './_variables.scss';
+
+@mixin flex {
+  display: flex;
+}
+
+@mixin flex-row {
```

Comment:
Why is this a mixin can be a placeholder type

Reply:
yes, I have kept both mixin and placeholder type for this. Because scss does not allow to extend under media query break-point and in those cases we could include these mixins if require.